src/it/resources/org/checkstyle/suppressionxpathfilter/superclone/SuppressionXpathRegressionSuperCloneInnerClone.java

```diff
@@ -0,0 +1,15 @@
+package org.checkstyle.suppressionxpathfilter.superclone;
+
+public class SuppressionXpathRegressionSuperCloneInnerClone {
+    public Object clone() // warn
+    {
```

Comment:
This is same as SuppressionXpathRegressionSuperClonePlainAndSubclasses
Make this method to non clone to get violation in inner class